packages/jest-cli/src/TestScheduler.js

```diff
@@ -29,8 +29,7 @@ import SummaryReporter from './reporters/summary_reporter';
 import TestRunner from 'jest-runner';
 import TestWatcher from './TestWatcher';
 import VerboseReporter from './reporters/verbose_reporter';
-
-const SLOW_TEST_TIME = 1000;
+import * as testSchedulerHelper from './testSchedulerHelper';
```

Comment:
I'd prefer to import by name, because why not:
```suggestion
import {computeRunInBand} from './testSchedulerHelper';
```

Reply:
i need this import to be able to write the test and use `spyOn`.
do you have an example on how to `spyOn` if i change the import?

Reply:
It's the same module, so it shouldn't matter 🤔

Reply:
https://stackblitz.com/edit/testing-import-vs-import?file=src%2Fapp%2Ftest%2Ftest.spec.ts
example online with jasmine since i don't have jest on stackblitz but it doesn't matter for one case :)

Reply:
I don't see how's that related. Check out my latest commit to this PR

Reply:
i feel really dumb now
it's working o O
https://stackblitz.com/edit/testing-import-vs-import-cqfvh3?file=src%2Fapp%2Ftest%2Ftest.spec.ts

Reply:
@thymikee for some reason i assumed it's  was failing by old experience... my bad it's all fine
Thanks for the fix (i correct something in my head :P)